src/components/Chatting/MessagesBody/ChatSet.jsx

```diff
@@ -6,6 +6,152 @@ import { getS3Url } from "tools/trans";
 import PropTypes from "prop-types";
 import theme from "styles/theme";
 import ChatPaySettle from "./ChatPaySettle";
+import WalletIcon from "@mui/icons-material/Wallet";
+import ContentCopyIcon from "@mui/icons-material/ContentCopy";
+import { height } from "@mui/system";
+import hoverEventSet from "tools/hoverEventSet";
+import CheckIcon from "@mui/icons-material/Check";
+
+const ChatAccount = (props) => {
+  const bankName = props.account.split(" ")[0];
+  const accounNumber = props.account.split(" ")[1];
+  const [clicked, setClicked] = useState(false);
+  const [isCopied, setIsCopied] = useState(false);
+
+  useEffect(() => {
+    if (isCopied) {
+      setTimeout(() => setIsCopied(false), 1000);
+    }
+  }, [isCopied]);
```

Comment:
따닥 클릭 방지 및 컴포넌트 unmount 대비 이유로 아래 처럼 바뀌면 좋을 것 같아요

```suggestion
  useEffect(() => {
    if (isCopied) {
      const timer = setTimeout(() => setIsCopied(false), 1000);
      return () => clearTimeout(timer);
    }
  }, [isCopied]);
```